plugins/statistics.koplugin/main.lua

```diff
@@ -1046,6 +1086,71 @@ The max value ensures a page you stay on for a long time (because you fell aslee
+                        callback = function(touchmenu_instance)
+                            local server = self.settings.sync_server
+                            local edit_cb = function()
+                                local sync_settings = SyncService:new{}
+                                sync_settings.onClose = function(this)
+                                    UIManager:close(this)
+                                end
+                                sync_settings.onConfirm = function(sv)
+                                    self.settings.sync_server = sv
+                                    touchmenu_instance:updateItems()
+                                end
+                                UIManager:show(sync_settings)
+                            end
+                            if not server then
+                                edit_cb()
+                                return
+                            end
+                            local dialogue
+                            local delete_button = {
+                                text = _("Delete"),
+                                callback = function()
+                                    UIManager:close(dialogue)
+                                    UIManager:show(ConfirmBox:new{
+                                        text = _("Delete server info?"),
+                                        cancel_text = _("Cancel"),
+                                        cancel_callback = function()
+                                            return
+                                        end,
+                                        ok_text = _("Delete"),
+                                        ok_callback = function()
+                                            self.settings.sync_server = nil
+                                            touchmenu_instance:updateItems()
+                                        end,
+                                    })
+                                end,
+                            }
+                            local edit_button = {
+                                text = _("Edit"),
+                                callback = function()
+                                    UIManager:close(dialogue)
+                                    edit_cb()
+                                end
+                            }
+                            local close_button = {
+                                text = _("Close"),
+                                callback = function()
+                                    UIManager:close(dialogue)
+                                end
+                            }
+                            local type = server.type == "dropbox" and " (DropBox)" or " (WebDAV)"
+                            dialogue = require("ui/widget/buttondialogtitle"):new{
+                                title = T(_("Cloud storage:\n%1\n\nFolder path:\n%2\n\nTo sync across devices, set them with the same Cloud folder."),
```

Comment:
What does this mean exactly?

Reply:
You mean the wording? It means to enable sync across devices, one should set them with the same remote path.

Reply:
Something like:
```suggestion
                                title = T(_("Cloud storage:\n%1\n\nFolder path:\n%2\n\nThe same remote path should be configured on all devices."),
```

But it's not clear to me where it's located and there were no screenshots. It's always a good practice to include a couple. ^_^

Reply:
<img width="250" alt="image" src="https://user-images.githubusercontent.com/59040746/201329493-d27ab318-1c1c-4590-a6b7-9de119faf270.png">

As this type of Cloud storage and folder path is shown above this sentence, it should be clear enough?

Reply:
Change it to something like this then:

```suggestion
                                title = T(_("Cloud storage:\n%1\n\nFolder path:\n%2\n\nSet up the same cloud folder on each device to sync across your devices."),
```

Reply:
Done.